signac/warnings.py

```diff
@@ -3,6 +3,9 @@
 # This software is licensed under the BSD 3-Clause License.
 
 
+"""Module for signac depercation warnings."""
```

Comment:
```suggestion
"""Module for signac deprecation warnings."""
```

Reply:
@atravitz @mikemhenry Why was this merged without this fix applied?